lldb/include/lldb/Target/StackFrame.h

```diff
@@ -324,8 +325,29 @@ class StackFrame : public ExecutionContextScope,
   ///    C string with the assembly instructions for this function.
   const char *Disassemble();
 
+  /// Print a description of this frame using the provided frame format.
+  ///
+  /// \param[out] strm
+  ///   The Stream to print the description to.
+  ///
+  /// \param[in] frame_marker
+  ///   Optional string that will be prepended to the frame output description.
+  ///
+  /// \param[in] use_fallback_format_on_error
+  ///   If dumping with the given \p format fails and this flag is enabled, then
+  ///   dumping will be retried with a default fallback format.
```

Comment:
Maybe change the wording so this will use the current `frame-format` from the settings? The term "fallback format" doesn't clearly indicate what is going to happen here. Maybe:
```
  /// \param[in] use_frame_format_from_settings_on_error
  ///   If dumping with the given \p format fails and this flag is enabled, then
  ///   dumping will be retried with the format from the "frame-format" setting.